### DISCUSSION TITLE
Date Picker not working on mobile devices

### DISCUSSION THREAD

**Discussion by berma83** [Budibase/budibase#2852] [Help] [answered]
Hi!

Built-in calendar dropdown is not working on mobile version website. See below:

https://user-images.githubusercontent.com/53205779/135633401-73ca5b04-d305-4670-80fc-ddd26c60a3ad.mp4

**Comment by mjashanks** [comment, marked as answer]
Thanks @berma83. i have reproduced this and logged as an issue: https://github.com/Budibase/budibase/issues/2878

Thanks for reporting!
